packages/styles/scss/patterns/leadspace/_leadspace.scss

```diff
@@ -5,11 +5,135 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import '../../globals/imports';
-
 @mixin leadspace {
   .#{$prefix}--leadspace {
-    background-color: transparent;
+    position: relative;
+    padding-top: carbon--rem(370px) / carbon--mini-units(40) * 100%;
+    height: 0;
+    background: $ui-background;
+
+    &__image {
+      object-fit: cover;
+      width: 100%;
+      height: 100%;
+    }
+
+    &__container {
+      position: absolute;
+      top: 0;
+      height: 100%;
+      width: 100%;
+    }
+
+    &__overlay {
+      padding: $layout-03 $layout-01;
+      position: absolute;
+      top: 0;
+      height: 100%;
+      width: 100%;
+      background-image: linear-gradient(
+        to bottom,
+        rgba($ui-background, 1),
+        transparent
+      );
+    }
+
+    &__content {
+      position: absolute;
+      bottom: $layout-03;
+    }
+
+    h1 {
+      @include carbon--type-style(display-01, true);
+      color: $text-01;
+    }
+
+    p {
+      @include carbon--type-style(expressive-heading-03, true);
+      color: $text-01;
+    }
+
+    &__ctas {
+      display: flex;
+      flex-direction: column-reverse;
+    }
+
+    button,
+    a {
+      width: 100%;
+    }
+
+    button:nth-child(2),
+    a:nth-child(2) {
+      margin: $layout-03 0 $layout-01 0;
+    }
+  }
+
+  .#{$prefix}--leadspace--productive h1 {
+    @include carbon--type-style(expressive-heading-06, true);
+  }
+
+  @include carbon--breakpoint(md) {
+    .#{$prefix}--leadspace {
+      padding-top: carbon--mini-units(50) / carbon--mini-units(84) * 100%;
+
+      &__overlay {
+        padding: $layout-03;
+        background-image: linear-gradient(
```

Comment:
Can you remove the following two properties if you do `background: linear-gradient(to right, $ui-background, transparent 50%);` here instead?

Reply:
yep